Python/errors.c

```diff
@@ -513,9 +531,16 @@ PyErr_SetExcInfo(PyObject *p_type, PyObject *p_value, PyObject *p_traceback)
     oldvalue = tstate->exc_info->exc_value;
     oldtraceback = tstate->exc_info->exc_traceback;
 
-    tstate->exc_info->exc_type = p_type;
```

Comment:
I can't put a comment on line 525, but it looks like a mistake that the arguments here have names starting with `p_`. For `PyErr_GetExcInfo()` that naming makes sense, because it has output parameters (i.e. *pointers* to `PyObject *`) but here I think it just was a copy-paste by the original author (or maybe a misunderstanding about the naming convention).